packages/roles/package.js

```diff
@@ -9,13 +9,16 @@ Package.describe({
 Package.onUse((api) => {
   api.versionsFrom('1.11.1');
 
+  Npm.depends({
+    'lodash.clone': '4.5.0',
+  });
+
   api.use([
     'unchained:core-users',
     'meteor-base',
     'check',
     'mongo',
     'ecmascript',
-    'underscore',
     'dburles:collection-helpers@1.1.0',
```

Comment:
The `check` package is used in many places in this packages. Since our main goal is to break free from Meteor, I can replace it with an NPM package instead.